core-tests/src/test/java/org/mule/runtime/core/DefaultMuleMessageTestCase.java

```diff
@@ -40,6 +40,8 @@
 public class DefaultMuleMessageTestCase extends AbstractMuleContextTestCase
 {
 
+    private static final String PAYLOAD = "MULE_MESSAGE";
```

Comment:
If you are going to update literal test payloads use `org.mule.tck.junit4.AbstractMuleContextTestCase#TEST_PAYLOAD`.